_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+
+# A git hook script that runs the same checks that are 
```

Comment:
Re-formatted comment to break closer to 80 columns.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/3f6000223f6819c18a5133a60e77369a825dff00